### ISSUE TITLE
Type error in handling parameters for gnt-os modify

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#367]
Originally reported of Google Code with ID 311.

```
[cluster] root@node4:~# gnt-os modify -H xen-pvm:kernel_path=/bin/true foo
(ok)
[cluster] root@node4:~# gnt-os modify -H -xen-pvm foo
Failure: command execution error:
'NoneType' object is not iterable

  File "/usr/lib/python2.6/dist-packages/ganeti/cmdlib.py", line 4123, in CheckPrereq
    self.new_os_hvp[os_name][hv_name].update(hv_dict)
TypeError: 'NoneType' object is not iterable

```
Originally added on 2012-11-15 16:18:57 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-12-12 10:31:30 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
commit 8e8092c
Author: Michael Hanselmann <hansmi@google.com>
Date:   Wed Dec 12 13:17:29 2012 +0100

    Fix TypeError when unsetting OS parameters
    
    When all OS parameters should be unset (“gnt-os modify -H -xen-pvm
    foo”), a TypeError was raised. This fixes issue 311.
    
    Signed-off-by: Michael Hanselmann <hansmi@google.com>
    Reviewed-by: Iustin Pop <iustin@google.com>

```
Originally added on 2012-12-12 13:34:11 +0000 UTC.

Mentioned Issue with GoogleCode ID 311 has been migrated to GitHub with ID #367.
Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking as released the bugs fixed in 2.6.2.
```
Originally added on 2013-01-11 12:13:05 +0000 UTC.

Changed State: Released
